mkdocs/docs/datastore/GET/contaovos.md

```diff
@@ -69,7 +69,7 @@ The request requires an API key provided by Contaovos moderation. To more inform
 === "curl"
     ```sh
     curl -X 'POST' \
```

Comment:
I've moved this page to "Retrieve" tab, but I won't update the POST method to GET until it's changed in the API